ale/drivers/cassini_drivers.py

```diff
@@ -175,3 +232,47 @@ def sensor_model_version(self):
           ISIS sensor model version
         """
         return 1
+
+    @property
+    def frame_chain(self):
+        if not hasattr(self, '_frame_chain'):
+            if self.instrument_id == 'CASSINI_ISS_NAC':
+                self._frame_chain = FrameChain.from_spice(frame_changes = [(1, self._original_naif_sensor_frame_id), (1, self.target_frame_id)], ephemeris_time=self.ephemeris_time)
+                rot_180 = Rotation.from_euler('z', 180, degrees=True)
+                self._frame_chain.add_edge(self._original_naif_sensor_frame_id, self.sensor_frame_id, ConstantRotation(rot_180.as_quat(), self._original_naif_sensor_frame_id, self.sensor_frame_id))
+            elif self.instrument_id == "CASSINI_ISS_WAC":
+                self._frame_chain =  super(CassiniIssPds3LabelNaifSpiceDriver, self).frame_chain
+        return self._frame_chain
+
+    @property
+    def focal_length(self):
+        """
+
+        """
+        # default focal defined by IK kernel
+        default_focal_len = super(CassiniIssPds3LabelNaifSpiceDriver, self).focal_length
+
+        if self.instrument_id == "CASSINI_ISS_NAC":
+          filters = tuple(self.label['FILTER_NAME'])
+          return self.nac_filter_to_focal_length.get(filters, default_focal_len)
+
+        elif self.instrument_id == "CASSINI_ISS_WAC":
+           return default_focal_len
+
+    @property
+    def _original_naif_sensor_frame_id(self):
+        """
+        Returns the Naif ID code for the target reference frame
+        Expects the target_id to be defined. This must be the integer Naif ID code
+        for the target body.
+
+        Returns
+        -------
+        : int
+          Naif ID code for the target frame
```

Comment:
These docs look incorrect

Reply:
all docs should be updated now for new methods